### DISCUSSION TITLE
callback while hovering through dropdown items

### DISCUSSION THREAD

**Discussion by Berkdan** [fomantic/Fomantic-UI#1932] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hi,
Is there any callback function while hovering through dropdown items? if not, is it possible to implement it?

Thanks

**Comment by lubber-de** [comment, marked as answer]
It's not built-in but you can implement it easily on your own
See https://jsfiddle.net/lubber/9ez0ofm3/5/

```javascript
$('.ui.dropdown').dropdown({});
$('.ui.dropdown .item ').hover(function(){
	console.log('You hovered over '+$(this).html())
})
```

**Comment by Berkdan** [comment]
Brilliant!
Thank you, that is what exactly i was looking for.
